njsscan/__main__.py

```diff
@@ -84,6 +84,65 @@ def json_output(outfile, scan_results):
         return json_output
 
 
+def sonarqube_output(outfile, scan_results):
+    """Sonarqube JSON Output."""
+    sonarqube_issues = []
+    for k, v in scan_results['nodejs'].items():
```

Comment:
The scan result also have findings from HTML templates `results['templates']` Is that of no interest?